ui/de-lib/src/main/java/org/iplantc/de/apps/client/models/AppCategoryStringValueProvider.java

```diff
@@ -11,7 +11,7 @@
 
     @Override
     public String getValue(AppCategory object) {
-        return object.getName() + " (" + object.getAppCount() + ")";
+        return object.getName();
```

Comment:
Yeah, I think we are leaving the counts out for now, since these categories can now display a mix of Agave and DE apps, and we don't currently have a count for Agave apps under each category.